pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -402,12 +411,16 @@ func (p *PriorityQueue) Pop() (*v1.Pod, error) {
 		}
 		p.cond.Wait()
 	}
+
 	obj, err := p.activeQ.Pop()
 	if err != nil {
 		return nil, err
 	}
 	pInfo := obj.(*framework.PodInfo)
 	p.schedulingCycle++
+
+	end := time.Now()
+	klog.V(2).Infof("Pod pod %v-$v, duration %v ms", pInfo.Pod.Namespace, pInfo.Pod.Name, end.Sub(start).Milliseconds())
```

Comment:
Pop pod?

Reply:
fixed. was fixed on my test box with the logging % instead of $, but not back add to the dev machine where the commit was made.